docker-compose.yml

```diff
@@ -95,6 +95,7 @@ services:
       - -dopcache.preload=opcache.preload=/usr/src/code/app/preload.php
     environment:
       - _APP_ENV
+      - _APP_HOME
```

Comment:
We'll also either need to hardcode the value here, or add it in `variables.php` with a default value

Reply:
@abnegate I had second thoughts about it since we also use it in the `specs.php` file where its used to generate some very specific URLs like our terms and conditions etc. 
https://github.com/appwrite/appwrite/blob/master/src/Appwrite/Platform/Tasks/Specs.php#L244

So it made sense it add a default inside the dockerfile and not override it.